app/Policies/JobPolicy.php

```diff
@@ -108,7 +108,7 @@ public function reviewApplicationsFor(User $user, JobPoster $jobPoster)
             return $jobPoster->isPublic() && ($authManager || $authHr);
         }
 
-        return $jobPoster->isClosed() && ($authManager || $authHr);
+        return $jobPoster->isPublic() && $authManager || $jobPoster->isClosed() &&  $authHr;
```

Comment:
I checked with Gray, we want to treat HR and Managers the same when it comes to job access. So we want `return $jobPoster->isPublic() && ($authManager || $authHr);`